conn_test.go

```diff
@@ -0,0 +1,51 @@
+package gorqlite_test
+
+import (
+	"testing"
+
+	"github.com/rqlite/gorqlite/v2"
+)
+
+func TestSetConsistencyLevel(t *testing.T) {
```

Comment:
How about using `conn.ConsistencyLevel()` to validate that the consistency level was actually changed?